src/main/app/case/from-api-format.ts

```diff
@@ -4,6 +4,7 @@ import { invert } from 'lodash';
 
 import { Case, CaseDate, Checkbox, formFieldsToCaseMapping, formatCase } from './case';
 import { CaseData, YesOrNo } from './definition';
+//import { fromHearingApiApplicant1 as hearingCollectionFromApiApplicant1 } from './formatter/hearings-list';
```

Comment:
remove commented out code if not required.

Reply:
removed the commented out code